models/stg_shopify.yml

```diff
@@ -1570,7 +1569,7 @@ models:
       - name: created_at
         description: When the checkout discount application was created.
       - name: discount_id
-        description: ID of the discount.
+        description: ID of the discount. Use `code` instead.
```

Comment:
```suggestion
        description: ID of the discount. Deprecated, use `code` instead.
```

Reply:
yeah i originally had this, but I couldn't confirm in the shopify docs that this field is specifically _deprecated_. i only found unanswered questions about the field in the forums, so might wanna keep it as is?

Reply:
I did see it's deprecated in the [Fivertran release notes](https://fivetran.com/docs/applications/shopify/changelog#july2021), so maybe that's enough confirmation? If not, makes sense to leave it as-is!